### PR TITLE
2753669 access cart anonymous

### PR DESCRIPTION
**Pull request by haringsrob**
I did not add specific tests, as this is covered by default.

### CODE REVIEW

File: modules/cart/commerce_cart.install [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * @file
+ * Contains install and update functions for Cart.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function commerce_cart_install() {
+  // Allow all roles to use cart.
+  user_role_grant_permissions('anonymous', ['access cart']);
```
Comment (mglaman) [inline modules/cart/commerce_cart.install]: `AccountInterface::ANONYMOUS` or however.

File: modules/cart/commerce_cart.install [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  user_role_grant_permissions('authenticated', ['access cart']);
```
Comment (mglaman) [inline modules/cart/commerce_cart.install]: `AccountInterface::AUTHENTICATED` or however.

File: modules/cart/commerce_cart.install (hunk 1 above)
Comment (mglaman) [inline modules/cart/commerce_cart.install]: Or `RoleInterface::ANONYMOUS_ID` as below O:)

File: modules/checkout/tests/src/Functional/CheckoutOrderTest.php [hunk 3]
```diff
@@ -64,11 +64,6 @@ protected function setUp() {
       'variations' => [$variation],
       'stores' => [$store],
     ]);
-
-    user_role_grant_permissions(RoleInterface::ANONYMOUS_ID, [
```
Comment (mglaman) [inline modules/checkout/tests/src/Functional/CheckoutOrderTest.php:68]: We have an unused import now, which is throwing phpcs error in Travis.

**Comment by haringsrob** [comment]
Alright, issues have been resolved.

File: modules/cart/tests/src/Functional/CartTest.php [hunk 4]
```diff
@@ -41,7 +43,6 @@ class CartTest extends OrderBrowserTestBase {
   protected function getAdministratorPermissions() {
     return array_merge([
       'administer products',
-      'access cart',
```
Comment (ndf) [inline modules/cart/tests/src/Functional/CartTest.php]: The permissions 'access cart' and 'access checkout' are also in /commerce/modules/checkout/tests/src/Functional/CheckoutOrderTest.php
~line 68

Can we remove them there to in this PR?

**Pull request merged**